tiatoolbox/tools/patchextraction.py

```diff
@@ -54,14 +62,22 @@ class PatchExtractor(ABC):
         pad_constant_values (int or tuple(int)): Values to use with
           constant padding. Defaults to 0. See :func:`numpy.pad` for
           more.
+        within_bound (bool): whether to extract patches beyond the
```

Comment:
what if it is False.
change if yes to if True

Reply:
```
within_bound (bool): Whether to include the patches on the right and bottom
                margins of mother image. If `True`, the patches that their location
                exceeds the `image_shape` would be neglected. Otherwise, those patches
                would be extracted with `Reader` function and appropriate padding.
```

I have changed the class docstrings accordingly.

Reply:
resolved in 137e25c758490f665c967a81f0263c34f5d17677